packages/ds-ext/src/DiagramEditorProvider.ts

```diff
@@ -59,7 +59,7 @@ export class DiagramEditorProvider implements vscode.CustomEditorProvider<Diagra
       };
 
       const handler = handlers[event.type];
-      if(!handler) throw Error(`No handler found for event type: ${event.type}. Available handlers: ${Object.keys(handlers).join(', ')}`);
```

Comment:
I think we should not enforce brackets (setting: curly 'warn'?) as I think no-brackets makes sense in some situations, like early exits for instance.

Reply:
I have updated the two suggestions in my code.